docusaurus/docs/Android/02-client/06-guides/11-video-agora.mdx

```diff
@@ -0,0 +1,798 @@
+# Agora: Video Integration Guide for Android
+
+This page shows you how to integrate [Agora](https://www.agora.io/en/) into Stream Chat SDK for implementing video call features.
+
+## Introduction
+Video calls have become immensely popular since the onset of the pandemic. Today, we take a look at how you can use the service of [Agora](https://www.agora.io/en/) to integrate video calls into the Stream Chat SDK.
+
+Agora is an infrastructure provider for live, interactive voice and video. They offer native SDKs for mobile platforms, cross-platform frameworks, and the web and allow for simple integration with very few lines of code. They cover a wide range of use cases such as video conferencing, interactive live-streaming, real-time messaging, and many more.
+
+You must complete quite a few steps to create the final product. We will cover all of them to help you create a well-integrated, fully functional, and reusable solution in the end.
+
+Follow the steps below to produce this app that allows your users to make video calls:
+
+1. Setting up an account for Agora
+2. Stream Dashboard integration
+3. Configure dependencies and permissions
+4. Implement a ViewModel for video calls
+5. Implement video call layouts
+6. Implement custom call messages using custom attachments
+
+:::note
+This tutorial assumes you already use [Stream Chat SDK for Android](https://getstream.io/chat/sdk/android/) in your Android project. So you will not cover how to set up the Stream Chat SDK in this tutorial. For more information, check out the [Android Chat Messaging Tutorial](https://getstream.io/tutorials/android-chat/) guide.
+:::
+
+## 1. Setting up an account for Agora
+
+You need to set up an account on the agora.io website. Once you’ve created the account, you will need to create a project and look for it in the console. Once you have that ready, you will need to prepare two things for the creation of the server in the next chapter:
+
+- App ID
+- App certificate
+
+Once you have these two available you can continue with this guide.
+
+## 2. Stream Dashboard integration
+
+In order for the integration of agora to work there needs to be a server component handling token generation and more. Normally this would require a custom server implementation but Stream offers first-class integration for agora relieving you of these duties.
```

Comment:
```suggestion
In order for the integration of **Agora** to work there needs to be a server component handling token generation and more. Normally this would require a custom server implementation, but **Stream** offers first-class integration for **Agora** relieving you of these duties.
```

Reply:
done: [6e70f22](https://github.com/GetStream/stream-chat-android/pull/4205/commits/6e70f220cf1d7c2124649b896c7536087a448eef)